docs/03-explore.md

```diff
@@ -60,19 +67,32 @@ ip-10-0-168-43.ec2.internal    Ready     master    23h       v1.12.4+5dc94f3fda
 ~~~
 
-For reference, **labels** are used as a mechanism to tag certain information onto a node, or a set of nodes, that can help you identify your systems, e.g. by operating system, system architecture, specification, location of the system (e.g. region), it's hostname, etc. They can also help with application scheduling, e.g. make sure that my application (or pod) resides on a specific system type. The labels shown above are utilising the default labels, but it's possible to set some custom labels in the form of a key-value pair.
+For reference, **labels** are used as a mechanism to tag certain information
+onto a node, or a set of nodes, that can help you identify your systems, e.g.
+by operating system, system architecture, specification, location of the system
+(e.g. region), it's hostname, etc. They can also help with application
+scheduling, e.g. make sure that my application (or pod) resides on a specific
+system type. The labels shown above are utilising the default labels, but it's
+possible to set some custom labels in the form of a key-value pair.
 
 ## The Cluster Operator
 
-The Cluster Operator is the most important implementation within OpenShift, it's a self-hosted operator that is heavily responsible for installation, management, maintenance, and automated operations on the OpenShift cluster; it's essentially what turns Kubernetes into OpenShift - it enables all of the additional operators that we rely on (including the web console, the marketplace, image registry, and router service, etc) and configures the cluster as we have specified. We can view it's status by using the following command-
+The Cluster Operator is the most important implementation within OpenShift, it's
+a self-hosted operator that is heavily responsible for installation, management,
+maintenance, and automated operations on the OpenShift cluster; it's essentially
+what turns Kubernetes into OpenShift - it enables all of the additional
+operators that we rely on (including the web console, the marketplace, image
+registry, and router service, etc) and configures the cluster as we have
+specified. We can view it's status by using the following command-
 
 ~~~bash
 $ oc get deployments -n openshift-cluster-version
 NAME                       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
 cluster-version-operator   1         1         1            1           2h
 ~~~
 
-You can also `rsh` (remote shell access) into the running Operator and see the various manifests associated with the installed release of OpenShift:
+You can also `rsh` (remote shell access) into the running Operator and see the
+various manifests associated with the installed release of OpenShift:
 
 ~~~bash
 $ oc rsh -n openshift-cluster-version deployments/cluster-version-operator
```

Comment:
This is useful only in a very narrow set of use cases.  It might be worthwhile to replace, or expand, this section with the command `oc adm release info --commits`.  This will output the payload of the cluster version operator, providing the GitHub repoistory and specific commit at the point where the cluster version was "created".

This can be extremely useful for finding additional information about operators because it provides the location of that specific operator's documentation and code.

Reply:
OK will add that in after I rebase. Thanks!

Reply:
@acsulli - I don't think that command is fully implemented in the beta2 drop env that I have access to.

```
rdo$ oc adm release info --commits
Error: unknown flag: --commits


Available Commands:
  extract     Extract the contents of an update payload to disk
  new         Create a new OpenShift release

Use "oc adm <command> --help" for more information about a given command.

```